src/content/docs/style-guide/writing-docs/article-templates/intro-branch-install-doc.mdx

```diff
@@ -0,0 +1,138 @@
+
+* The "default content" approach relies on using the `override` and `skip` components. It displays a file and then says: skip or override this file when x options are chosen. (Install doc that uses this: [the Java install](/install/java).)
+* If you don't want to display any content before options are selected, you'd use the `isCondition=true` approach. That component says simply: "display this file if these options are selected." (Install doc that uses this: [the .NET install](/install/dotnet).)
+  </Collapser>
+
+  <Collapser
+    id="display-all-options-decision"
+    title="Display all options? Or nest them?"
+  >
+This decision is about whether you want to initially display all the option choices, or if you'd rather nest them (for example, choosing one option results in another option selection box).
+
+If your option selection logic is fairly simple, it can make sense to simply expose all the options to customers at once. However, if your option selection logic is rather complex (for example, three or more selections, and some selections affecting later selections), it can make sense to nest them. 
+
+For an example of a fully exposed set of option selection boxes, see the [Network security prerequisites doc]. For an example of a nested set of selection boxes, see [the .NET install].
+  </Collapser>
+
+  <Collapser
+    id="guided-install-decision"
+    title="Guided install?"
+  >
+
+This decision point is about whether you will need to display information about our guided install path. For some agents, we want to display some content about the guided install path (for example, you can see the guided install content in the [Java agent install](/install/java) if you select "app server" and "Jetty"). 
+
+We have a couple options for adding the guided install content: 
+
+* We have a `recommendedGuided: true` component that you can place on specific options where you want the guided install content to show up. What this essentially does is displays the content of the .mdx file that contains that option selection box. You can see an example of this in the Java install, where the guided install content is located in the `appInfo-2` file. This implementation isn't actually specific to guided install content: you could use this logic to display any arbitrary content you might want to use. 
+* Alternatively, you could just add guided install as a regular .mdx file and display it as you would any other file. The .NET install takes this approach. With this approach it will appear as a numbered step like other files. 
+
+  </Collapser>
+
+  <Collapser
+    id="config-file-decision"
+    title="Config file?"
```

Comment:
Maybe change all these collapsers to full sentences/questions, like "Do I need a config file"?